cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -132,6 +132,8 @@ object AnalyzerCommand : CommandWithHelp() {
         val ortResult = analyzer.analyze(absoluteProjectPath, packageManagers, packageCurationsFile,
                 repositoryConfigurationFile)
```

Comment:
commit message: 

typo "[..]  whether a result `is` empty."

Reply:
Done.